src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import absolute_import
+
+from django.core.urlresolvers import reverse
+
+from sentry import http, options
+from sentry.api import client
+from sentry.api.base import Endpoint
+from sentry.models import Group, Integration, Project, IdentityProvider, Identity
+from sentry.utils import json
+from sentry.utils.http import absolute_uri
+
+from .utils import build_attachment, logger
+
+LINK_IDENTITY_MESSAGE = "Looks like you haven't linked your Sentry account with your Slack identity yet! <{link_url}|Link your identity now> to perform actions in Sentry through Slack."
+
+RESOLVE_SELECTOR = {
```

Comment:
If we want, we could also add another field to the resolve dialog for selecting a *specific* version, though we'd have to populate this list prior, since the dynamic list search does not work in dialogs.